src/hotspot/share/runtime/objectMonitor.hpp

```diff
@@ -148,7 +148,7 @@ class ObjectMonitor : public CHeapObj<mtInternal> {
   // Used by async deflation as a marker in the _owner field:
   #define DEFLATER_MARKER reinterpret_cast<void*>(-1)
   void* volatile _owner;            // pointer to owning thread OR BasicLock
-  volatile jlong _previous_owner_tid;  // thread id of the previous owner of the monitor
+  volatile uintptr_t _previous_owner_tid;  // thread id of the previous owner of the monitor
```

Comment:
Since you are changing the type for `_previous_owner_tid`, you also
need to change the `DEFINE_PAD_MINUS_SIZE` below:

```
  DEFINE_PAD_MINUS_SIZE(1, OM_CACHE_LINE_SIZE, sizeof(void* volatile) +
                        sizeof(volatile uintptr_t));
```

Reply:
Okay, thank you for pointing this out.